### DISCUSSION TITLE
IoTDB 数据库升级，数据迁移

### DISCUSSION THREAD

**Discussion by StrongManLi** [apache/iotdb#11472] [Q&A] [answered]
最近在进行 IoTDB 版本升级，需要进行数据迁移。
1.1.X版本的数据能直接同步到1.2.X
但是0.12版本和0.13版本的没找到相关的内容。有大佬能指导一下吗？大概需要那些步骤？

**Comment by HTHou** [comment, marked as answer]
https://iotdb.apache.org/zh/Download/ 官网下载页面有介绍具体步骤。

**Reply by StrongManLi** [reply]
谢谢
